<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Ankit-PC" name="Test" tests="2" failures="0" timestamp="30 Mar 2019 17:23:45 GMT" time="0.677" errors="0">
  <testcase name="@BeforeMethod beforeClass" time="0.0" classname="automationFramework.TestNg1">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at automationFramework.TestNg1.beforeClass(TestNg1.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod beforeClass -->
  <testcase name="@AfterMethod afterClass" time="0.0" classname="automationFramework.TestNg1">
    <skipped/>
  </testcase> <!-- @AfterMethod afterClass -->
  <testcase name="@BeforeMethod beforeClass" time="0.0" classname="automationFramework.TestNg__2">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at automationFramework.TestNg__2.beforeClass(TestNg__2.java:53)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod beforeClass -->
  <testcase name="@AfterMethod afterClass" time="0.0" classname="automationFramework.TestNg__2">
    <skipped/>
  </testcase> <!-- @AfterMethod afterClass -->
  <testcase name="f" time="0.009" classname="automationFramework.TestNg1">
    <skipped/>
  </testcase> <!-- f -->
  <testcase name="f" time="0.012" classname="automationFramework.TestNg__2">
    <skipped/>
  </testcase> <!-- f -->
</testsuite> <!-- Test -->
